### DISCUSSION TITLE
Creating Gutenberg blocks for all our widgets

### DISCUSSION THREAD

**Discussion by olatechpro** [publishpress/publishpress-taxonomies#1488] [Feature Requests]
Hi @stevejburge, as wordpress eventually deprecate old widget system in favour of blocks in wp 5.8 which shows it'll be remove in the future.

I think we need to create gutenberg blocks for all our previous widget and then deprecate all our legacy widgets or what do you think ?

You can decide on which appropriate milestone this will go to.

Right now, i'm fixing all the error with the legacy widget in wordpress 5.8 in another milestone since this is not an urgent issue.

**Comment by stevejburge** [comment]
@olatechpro I think we're OK for now, but yes ... we can do this in the future. Right now, we can do this:
<img width="353" alt="gutenberg-related-posts" src="https://user-images.githubusercontent.com/3868028/125635485-ca03bbed-933b-42aa-aea9-c07c51277ca1.png">

**Comment by olatechpro** [comment]
Cool
